### PR TITLE
[modbus] Fixed README.md for docs site

### PR DESCRIPTION
**Pull request by watou**
@ssalonen The Markdown renderer at docs.openhab.org is less forgiving than the one here, so I made some edits that should result in a legible and linkable version [here](http://docs.openhab.org/addons/bindings/modbus1/readme.html#writing-data).

Please update this file further with any outstanding PRs (and copy/paste to the old wiki if you want).

Thank you!

Signed-off-by: John Cocula <john@cocula.com>

### CODE REVIEW

File: bundles/binding/org.openhab.binding.modbus/README.md [hunk 1]
```diff
@@ -67,59 +75,26 @@ where:
...
+| start    | optional  | address of first coil/discrete input/register to read. Default is 0. The address is directly passed to the corresponding Modbus request, and thus is zero-based. See below for more information on the addressing. |
+| length   | mandatory | number of _data items_ to read. _Data items_ here refers to registers, coils or discrete inputs depending on the slave type. For example, if the goal is to read one item with `valuetype=int32`, one needs to read two registers (2 * 16bit = 32bit), thus `length = 2`. If three coils are of interest, one should specify `length = 3` |
+| valuetype | optional | tells how interpret the register data. For details, consult [Register interpretation (valuetype) on read & write](#register-interpretation-valuetype-on-read--write). | updateunchangeditems | optional | **Since 1.9.0*. `true` or `false`. Controls whether the binding sends an update event on every successful poll (`true`) or only if the state of the item actually changes (`false`).  Default is `false`. When polling many items with high poll frequency, setting this parameter to `true` may cause significant CPU usage. |
```
Comment (ssalonen) [inline bundles/binding/org.openhab.binding.modbus/README.md]: newline required before `| updateunchangeditems | ...` ?

File: bundles/binding/org.openhab.binding.modbus/README.md [hunk 2]
```diff
@@ -549,137 +522,42 @@ xxx.connection=/dev/pts/8:38400:8:none:1:rtu
...
-tcp.slave1.connection=localhost:55502:0:0:0:1
-tcp.slave1.type=holding
-tcp.slave1.length=20
-tcp.slave1.postundefinedonreaderror=true
-```
-
-As inspiration I used [Roller shutter wiki page](https://github.com/openhab/openhab1-addons/wiki/Rollershutter-Bindings)
-
-[Table of Contents](#table-of-contents)
+3. record logs and configuration to [pastebin.com](http://pastebin.com/).  The page contains examples for use with the [[Modbus binding|Modbus-Binding]].
```
Comment (ssalonen) [inline bundles/binding/org.openhab.binding.modbus/README.md]: something weird on this line:
- "The page contains examples ..." should be separate paragraph?
- perhaps the latter link is broken?

**Comment by ssalonen** [comment]
Thanks a lot. 

I'm been a bit out-of-loop of the documentation effort. So this is for openhab2, right?

Will the old wikis will remain for openhab1, or what is the plan?

Best,
Sami

**Comment by watou** [comment]
Hi Sami, Thanks for catching those errors.  I will fix them in a few minutes.  

> I'm been a bit out-of-loop of the documentation effort. So this is for openhab2, right?

Yes, for openHAB 2+ users of 1.x add-ons.  This README.md is reflected [here](http://docs.openhab.org/addons/bindings/modbus1/readme.html).

> Will the old wikis will remain for openhab1, or what is the plan?

At least I don't plan on removing the wiki pages! 😄  They only remain for 1.x runtime users.

The site docs.openhab.org means to encompass all of the official documentation for openHAB 2 onwards, which includes all the 1.x add-ons, but the 1.x wiki is full of references to openHAB 1.x installation and configuration information which can be confusing and misleading for openHAB 2 users.  So all 1.x add-on docs (around 189 of them) are now folded into docs.openhab.org via README.md files from the source, but made consistent with openHAB 2 (see PR #5029).  The wiki will still stay for openHAB 1.x runtime users, but it now shows a "warning" on the main page for openHAB 2 users to look to docs.openhab.org.  As the number of 1.x runtime users decreases and 2.x users increases, it seemed like the right time to unify the documentation, and to do so in a way that upset things as little as possible.

This means that it's up to whoever sees fit to update the wiki, as before, but the "official" documentation is generated from README.md files for each add-on.  An add-on's wiki page could be changed to note 1.x differences and a link to the page at docs.openhab.org just to simplify maintenance, or whatever seems to make sense.

**Comment by ssalonen** [comment]
Thank you very much opening the ongoing work. Will update and the old wiki with the official one.

Best
Sami

**Pull request merged**